src/utils/networks/types/MOONRIVER.js

```diff
@@ -0,0 +1,27 @@
+import tokens from '@/_generated/tokens/tokens-moonriver.json';
+import contracts from '@/_generated/contracts/contract-abi-moonriver.json';
+import moonriver from '@/assets/images/networks/moonriver.svg';
+
+export default {
+  name: 'Moonriver',
+  name_long: 'Moonriver',
+  homePage: 'https://moonriver.network/',
+  blockExplorerTX: 'https://moonscan.io/tx/[[txHash]]',
```

Comment:
Block explorer for moonriver is https://moonriver.moonscan.io/tx/
Current URL will search on moonbeam

Reply:
good catch